podman/tests/integration/test_manifests.py

```diff
@@ -0,0 +1,65 @@
+import unittest
+from contextlib import suppress
+
+from podman import PodmanClient
+from podman.errors.exceptions import APIError, ImageNotFound
+from podman.tests.integration import base
+
+# @unittest.skipIf(os.geteuid() != 0, 'Skipping, not running as root')
+
+
+class ManifestsIntegrationTest(base.IntegrationTest):
+    def setUp(self):
+        super().setUp()
+
+        self.client = PodmanClient(base_url=self.socket_uri)
+        self.addCleanup(self.client.close)
+
+        self.alpine_image = self.client.images.pull("quay.io/libpod/alpine", tag="latest")
+
+    def tearDown(self) -> None:
+        self.client.images.remove(self.alpine_image, force=True)
+        with suppress(ImageNotFound):
+            self.client.images.remove("quay.io/unittest/alpine:latest", force=True)
```

Comment:
Should this be `"quay.io/libpod/alpine:latest"`

Reply:
In fact...instead of duplicating, how about using object variables?

```python
class ManifestsIntegrationTest(base.IntegrationTest):
    TEST_IMAGE_NAME = "quay.io/libpod/alpine"
    TEST_IMAGE_TAG = "latest"
    ...
                self.client.images.remove(f"{self.TEST_IMAGE_NAME}:{self.TEST_IMAGE_TAG}", force=True)
```

Reply:
oh...nevermind...I understand what you're doing here.